doc/admin/installation/manual_smallscale.rst

```diff
@@ -120,6 +120,7 @@ Now we will install pretix itself. The following steps are to be executed as the
 actually install pretix, we will create a virtual environment to isolate the python packages from your global
 python installation::
 
+    $ sudo su pretix
```

Comment:
Thanks. I see and I understand; my pr was basically meant as soft reminder for upraders: especially when upgrading it is important to act as user `pretix`.


If you prefer - I would like that  - you can simply dismiss my pr and perhaps you can find a better solution to remind upgraders to act as user `pretix`.

Reply:
I think the explicit user switch is a good idea, but the more logical version that provides continuity to the previous command of the tutorial would be

```suggestion
    # su pretix
```

or if we prefer sudo

```suggestion
    # sudo -u pretix -s
```

I'd also get rid of the `deactive` since it seems distracting, we could try `exit` (if we assume sudo/su was used above) but could also be misleading?